File: answers/conditional asgnmt/Q3.c
#include <stdio.h>
#include <string.h>  
void main()
{  
   int custid, conu;
   float chg, surchg=0, gramt,netamt;
   char connm[25];

   printf("Input Customer ID :");
   scanf("%d",&custid);
   printf("Input the name of the customer :");
   scanf("%s",connm);
   printf("Input the litres consumed by the customer : ");
   scanf("%d",&conu);
   if (conu <199 )
	chg = 3;
   else	if (conu>=200 && conu<400)
		chg = 3.35;
	else if (conu>=400 && conu<600)
			chg = 4.80;
		else
			chg = 5.10;
   gramt = conu*chg;
   if (gramt>1250)
	surchg = gramt*15/100.0;
   netamt = gramt+surchg;
   if (netamt  < 100)
	netamt =100;
   printf("\nWater Bill\n");
   printf("Customer IDNO                       :%d\n",custid);
   printf("Customer Name                       :%s\n",connm);
   printf("litres Consumed                       :%d\n",conu);
   printf("Net Amount Paid By the Customer     :%8.2f\n",netamt);

}  

File: answers/VrishtiSharma/Conditonals/Q3.py
a = int(input("Enter Customer ID : "))
b = input("Enter the name of the customer : ")
c = int(input("Enter the litres consumed by the customer : "))
if c <= 199:
    s = c * 3
elif 200 <= c < 400:
    s = c * 3.35
elif 400 <= c < 600:
    s = c * 4.80
else:
    s = c * 5.10
if s > 1250:
    s = s + (15/100 * s)
print(a)
print(b)
print(c)
print(int(s))
